db/new_seeds/base/add_schools_for_cohortless_testing.rb

```diff
@@ -4,10 +4,10 @@ def padded_urn(num)
   num.to_s.rjust(6, "0")
 end
 
-cohort_22 = Cohort.find_by(start_year: 2022) || FactoryBot.create(:seed_cohort, start_year: 2022)
-cohort_23 = Cohort.find_by(start_year: 2023) || FactoryBot.create(:seed_cohort, start_year: 2023)
-FactoryBot.create(:ecf_schedule, cohort: cohort_23) if Finance::Schedule::ECF.default_for(cohort: cohort_23).blank?
-
+previous_cohort = Cohort.previous
+current_cohort = Cohort.current
+next_cohort = Cohort.next || FactoryBot.create(:seed_cohort, start_year: current_cohort.start_year + 1)
```

Comment:
ah I was thinking we avoid the next cohort altogether, and just keep previous and current cohorts and stick to the current setup? happy to chat through this tomorrow